### DISCUSSION TITLE
Can it run 'standalone'

### DISCUSSION THREAD

**Discussion by christau** [jomjol/AI-on-the-edge-device#2520] [Q&A]
Hi All,
I want to use this great piece of software in an environment that is not in the range of my wifi.
So my idea is basically to use it as a logger which opens up a wifi AP and provides its data via REST if I log on to this AP and do a REST call.
It should take a picture every day, store it to the sd card and do its 'magic' by decoding this picture and add this data to the internal storage.
Maybe I will get there twice a year and then download the historical data and images.

Is this possible?
I searched the docs and disussions so far but found no real help. Maybe I used the wrong terms.
Would be nice if someone could help me out with that.
Many thanks in advance.

**Comment by caco3** [comment]
You can run it without a network connection and store the data on the SD card. But I doubt you will be happy.

The recognition is (by its nature) not 100% save. There can be false positive which will mess your readings up.
